api/src/main/java/org/openmrs/module/coreapps/CoreAppsConstants.java

```diff
@@ -58,4 +58,6 @@
    public static final String GLOBAL_PROPERTY_CONDITIONS_CRITERIA = "coreapps.conditionListClasses";
 
    public static final String DEFAULT_CODING_SOURCE = "ICD-10-WHO";
+
+   public static final String DECEASED_DATE_USING_TIME = "coreapps.deceasedDateUsingTime";
```

Comment:
Can you use the same naming convention like the rest of the global property constants in this class?

Reply:
Sure i will change it.